### PR TITLE
enable parallel execution of web tests

### PR DESCRIPTION
**Pull request by stas-vilchik**
based on https://github.com/SonarSource/sonarqube/pull/83

### CODE REVIEW

File: server/sonar-web/src/main/js/tests/e2e/lib.js [hunk 1]
```diff
@@ -42,6 +42,11 @@ exports.changeWorkingDirectory = function (dir) {
 };
 
 
+exports.configureCasper = function () {
+  casper.options.waitTimeout = 30000;
```
Comment (jblievremont) [inline server/sonar-web/src/main/js/tests/e2e/lib.js]: Is this to avoid the 5 second timeout on checks?

Comment (stas-vilchik) [inline server/sonar-web/src/main/js/tests/e2e/lib.js]: this is to increase the timeout and avoid timeout failures

File: server/sonar-web/src/main/js/tests/e2e/tests/source-viewer-spec.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        test.assertSelectorContains('.source-viewer-header-measure', '6');
+      })
+
+      .run(function () {
+        test.done();
+      });
+});
+
+
+// FIXME enable test
```
Comment (simonbrandhof) [inline server/sonar-web/src/main/js/tests/e2e/tests/source-viewer-spec.js]: FIXME ou KILLME :-)

**Comment by simonbrandhof** [comment]
LGTM

**Comment by jblievremont** [comment]
LGTM (stable over 20 runs on my Linux box).

**Pull request merged**